tests/kernel/run_ltp.pm

```diff
@@ -242,11 +242,16 @@ sub run {
     my ($self) = @_;
     my $cmd_file = get_var 'LTP_COMMAND_FILE';
     die 'Need LTP_COMMAND_FILE to know which tests to run' unless $cmd_file;
+    my $conf_file   = '/root/env-variables.sh';                 # TODO: remove duplicity with ltp_setup_networking.pm
     my $cmd_pattern = get_var('LTP_COMMAND_PATTERN') || '.*';
     my $cmd_exclude = get_var('LTP_COMMAND_EXCLUDE') || '$^';
-    my $timeout     = get_var('LTP_TIMEOUT')         || 900;
+    my $timeout     = get_var('LTP_TIMEOUT') || 900;
     my $is_posix    = $cmd_file =~ m/^\s*openposix\s*$/i;
 
+    if ($conf_file) {
+        assert_script_run(". '$conf_file'");
```

Comment:
As far as I can tell this is always going to run and break the test if setup network testing is not ran. Also it might be simpler in the long run to set it as a start up script in ltp_setup_networking. I have a couple of minor issues with the setup_env_variables function, but I won't get into details until we have it mostly working.

BTW you probably mean 'duplication' not 'duplicity'.